mule-packager/src/main/java/org/mule/tools/api/verifier/ProjectVerifyFactory.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Mule ESB Maven Tools
+ * <p>
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * <p>
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.tools.api.verifier;
+
+import static org.mule.tools.api.packager.packaging.PackagingType.MULE_POLICY;
+
+import org.mule.tools.api.packager.ProjectInformation;
+import org.mule.tools.api.packager.packaging.PackagingType;
+import org.mule.tools.api.verifier.policy.MulePolicyVerifier;
+
+public class ProjectVerifyFactory {
+
+  public static ProjectVerifier create(ProjectInformation defaultProjectInformation) {
+
+    if (PackagingType.fromString(defaultProjectInformation.getPackaging()).equals(MULE_POLICY)) {
+      return new MulePolicyVerifier(defaultProjectInformation);
+    }
+
+    return new MuleProjectVerifier();
```

Comment:
Shouldn't this object be something like "NoProjectVerifier()" or "NullProjectVerifier()"?

Reply:
not really if I follow how this was done for the other phases.